content/_data/changelogs/weekly.yml

```
@@ -12426,6 +12427,62 @@
   # pull: 5681 (PR title: Bump spotless-maven-plugin from 2.12.2 to 2.12.3)
   # pull: 5695 (PR title: Update at-since up to 2.308)
 
+  - version: '2.310'
+    date: 2021-09-06
+    changes:
+      - type: TODO
+        category: developer
+        pull: 5703
+        authors:
+          - dependabot[bot]
+        pr_title: Bump jnr-posix from 3.1.7 to 3.1.8
+        message: |-
+          (No proposed changelog)
+      - type: rfe
+        category: rfe
+        pull: 5699
+        issue: 23784
+        authors:
+          - basil
+        pr_title: "[JENKINS-23784] - Register `DependencyClassLoader` in the default\
+          \ plugin strategy as parallel-capable"
+        message: |-
+          JENKINS-23784: Load classes from plugins in parallel for faster startup on multicore machines.
+      - type: rfe
+        category: rfe
+        pull: 5697
+        authors:
+          - janfaracik
+        pr_title: Graphs now scale correctly for high-DPI screens
+        message: |-
+          Graphs can now scale correctly to retina screens
+      - type: bug
+        category: bug
+        pull: 5704
+        authors:
+          - jonsten
+        pr_title: Fix bug in TextparameterDefinition
+        message: |-
+          Fix wrong parameter type for <code>Text Parameter</code> when triggering a build via the buildWithParameters API call
+      - type: bug
+        category: bug
```

The tool that @dheerajodha and @MarkEWaite run to generate the changelog would list this as a major bug.  We think that is because it is labeled `bug` and `regression-fix`.  Daniel did not label it `major bug`, so we assume it more accurate to list it as a `bug` than as a `major bug`.

No change in the suggestion
```suggestion
      - type: bug
        category: bug
```